app/src/main/java/openfoodfacts/github/scrachx/openfood/views/adapters/NutrimentsRecyclerViewAdapter.java

```diff
@@ -0,0 +1,96 @@
+package openfoodfacts.github.scrachx.openfood.views.adapters;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import java.util.List;
+
+import openfoodfacts.github.scrachx.openfood.R;
+import openfoodfacts.github.scrachx.openfood.models.NutrimentItem;
+
+/**
+ * @author herau
+ */
+public class NutrimentsRecyclerViewAdapter extends RecyclerView.Adapter {
+
+    private static final int TYPE_HEADER = 0;
+    private static final int TYPE_ITEM = 1;
+
+    private final List<NutrimentItem> nutrimentItems;
+
+    public NutrimentsRecyclerViewAdapter(List<NutrimentItem> nutrimentItems) {
+        super();
+        this.nutrimentItems = nutrimentItems;
+    }
+
+    @Override
+    public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        if (viewType == TYPE_ITEM) {
```

Comment:
You can also write it like that : https://github.com/openfoodfacts/OpenFoodFacts-androidApp/blob/master/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/adapters/ProductsRecyclerViewAdapter.java#L41 
It's only to be homogeneous, but I'm fine with that 👍